### ISSUE TITLE
Score changelog where sections cannot be linked to version

### ISSUE THREAD

**Issue by jonasfj** [dart-lang/pana#994]
https://pub.dev/packages/flutter_local_notifications/changelog

 * [ ] We should document how to write changelog
 * [ ] We should create a package for parsing changelogs like `changelog_parse`
 * [ ] We should score packages based on whether or not their changelog honors our spec.

**Comment by sigurdm** [comment]
Spec could contain something like:

* Headline l2 for each version. The headline should contain the version number (matching a regexp).

Do we require each version to have an entry?
 - only non-prereleases?
 - only the latest version?
Do we require each entry to be a published version?

**Comment by sigurdm** [comment]
Still relevant
Depending on #1406

**Comment by sigurdm** [comment]
Closing this in favor of https://github.com/dart-lang/pana/issues/1406
